src/docs/sphinx/quickstart.rst

```diff
@@ -145,7 +145,7 @@ Some helpful uberenv options include :
 
 * ``--spec=+debug`` (build the MFEM and Hypre libraries with debug symbols)
 * ``--spec=+glvis`` (build the optional glvis visualization library)
-* ``--spec=+caliper`` (build the `Caliper performance profiling library <https://github.com/LLNL/Caliper>`_)
+* ``--spec=+profiling`` (build the `Caliper performance profiling library <https://github.com/LLNL/Caliper>`_)
```

Comment:
This also builds Adiak and would be long in the tooth at that point with both links... 
```suggestion
* ``--spec=+profiling`` (build the Adiak and Caliper)
```

Reply:
Done in [8abf0289](https://github.com/LLNL/serac/pull/652/commits/8abf0289293cdb895311d516f2abb902fe13c6fa).